storage/aptosdb/src/metrics.rs

```diff
@@ -137,6 +137,23 @@ pub static ROCKSDB_PROPERTIES: Lazy<IntGaugeVec> = Lazy::new(|| {
     .unwrap()
 });
 
+// Async committer gauges:
+pub(crate) static LATEST_SNAPSHOT_VERSION: Lazy<IntGauge> = Lazy::new(|| {
+    register_int_gauge!(
+        "aptos_latest_snapshot_version",
```

Comment:
"state_snapshot"

also I think it's good to keep aptos_storage_ prefix to match other metrics in this crate